### DISCUSSION TITLE
Request for Information on Deleting Uploaded Files in PrimeNG Upload Component"

### DISCUSSION THREAD

**Discussion by prads46** [primefaces/community#1795] [PrimeNG] [answered]
I am currently facing an issue with the PrimeNG Upload component in my Angular application. Specifically, I am seeking guidance on how to implement functionality to delete files that have been uploaded using the PrimeNG Upload component.

The challenge I am encountering is that once a file has been uploaded, I am unable to find a built-in method or feature to delete that uploaded file. I have explored the documentation and searched for solutions online, but I have not been able to find a clear explanation or example on how to achieve this.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You can remove the file in advanced mode https://primeng.org/fileupload#advanced
